suite/auto-sync/src/autosync/cpptranslator/patches/GetOperandRegImm.py

```diff
@@ -12,7 +12,7 @@ class GetOperandRegImm(Patch):
     Patch   OPERAND.getReg()
     to      MCOperand_getReg(OPERAND)
 
-    Same for isImm()
+    Same for getImm()|getExpr
```

Comment:
So far I excluded support for expressions from Capstone. Because they were used for symbols, which is out of scope for Capstone. Is this different in Xtensa?